tgext/utils/storage.py

```diff
@@ -0,0 +1,32 @@
+import random
+import string
+from os import path, mkdir
+
+from tg import config
+
+
+def store(data):
+    extension = tell_extension(data)
+    filename = generate_id(extension)
+    destination = _specify_path(filename)
+
+    with open(destination, mode='wb') as dest:
+        content = data.file.read()
+        dest.write(content)
+        return filename
+
+
+def _specify_path(filename):
+    storage_path = path.join(config['paths']['static_files'], '..', 'storage')
```

Comment:
Why `..`? I thought the purpose was to store the files in public directory itself